packages/puppeteer/src/axePuppeteer.ts

```diff
@@ -17,7 +17,7 @@ interface IInjectAxeArgs {
   args?: any[];
 }
 
-function injectJSModule(frame: Frame): Promise<void> {
+function injectJSModule(frame: Frame): Promise<ElementHandle<Element> | void> {
```

Comment:
Might want to do the same hack as below to avoid this weird/useless `return` type:

```js
return frame.addScriptTag({ /* ... */ )}.then(() => undefined)
```